.github/workflows/periodic-merge.yml

```diff
@@ -0,0 +1,39 @@
+name: Periodic Merge
+
+on:
+  schedule:
+    - cron: 1,6,11,16,21,26,31,36,41,51,56 * * * *
```

Comment:
Just curious, why not use `*/5 * * * *` to say run every 5 minutes. Is there any particular reason to run at these exact minutes?

Reply:
Uh I read somewhere that scheduling it with some kind of offset is a bit better since usually there's an influx of jobs that get run on specific intervals like every 5 minutes. I found some cron expressions that did allow that but the yml validator didn't like them :| . I'm not opposed to changing it to that though.

Reply:
My experience with this is that using fixed offset makes it easier to know for sure when the action will be run, for example trying to find the corresponding logs using timestamp. It does make the cron a bit longer here, but that shouldn't be an issue. We probably only need to change if we somehow decide to run for every minute for example.